src/pal/inc/pal.h

```diff
@@ -484,7 +484,7 @@ BOOL
 PALAPI
 PAL_NotifyRuntimeStarted(VOID);
 
-static const int MAX_DEBUGGER_TRANSPORT_PIPE_NAME_LENGTH = 64;
```

Comment:
I think this was 64 chars because of limitations on OS X or some Linux distro. I might be mis-remembering this, but I do know that named semaphores on OS X do have a length limit.

Reply:
We just need to check on what OS X max pipe name length is.

Reply:
Hi @mikem8361 !
I did some digging for the osx pipe name length and couldn't find a size limit of 64. Maybe I missed something, here is what I found  : 
* https://opensource.apple.com/source/xnu/xnu-1699.24.8/bsd/kern/posix_sem.c
sem_open set pathlen to MAXPATHLEN
* https://opensource.apple.com/source/xnu/xnu-1699.24.8/bsd/sys/param.h.auto.html
MAXPATHLEN is set to PATH_MAX
* https://opensource.apple.com/source/xnu/xnu-1699.24.8/bsd/sys/syslimits.h.auto.html
#define	PATH_MAX		 1024

I tried to find more details in the history of the coreclr repo and I think that the value was first introduced in this PR : https://github.com/mikem8361/coreclr/commit/c240f969e0f43ee25f9590f7e53676e4fd7bd85d#diff-c548a2f48a115d3e692501836e81509aR83

But I couldn't find why it was set to 64.

Could you please help ? Thanks !